sources/en/l/literotica.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+import logging
+from typing import List
+
+from lncrawl.core.crawler import Crawler
+from lncrawl.models import Chapter, SearchResult
+
+logger = logging.getLogger(__name__)
+
+
+class LiteroticaCrawler(Crawler):
+    base_url = ["https://www.literotica.com/"]
+
+    def initialize(self) -> None:
+        self.init_executor(ratelimit=2)
+
+    def search_novel(self, query) -> List[SearchResult]:
+        pass
+        soup = self.get_soup(
+            f"https://search.literotica.com/?query={query}",
+            timeout=50)
+        results = []
+        for item in soup.select("div.panel.ai_gJ > div.ai_iG > a.ai_ii"):
+            results.append(
+                SearchResult(
+                    title=item.text.strip(),
+                    url=item["href"]
+                )
+            )
+        return results
+
+    def read_novel_info(self) -> None:
+        soup = self.get_soup(self.novel_url.replace("www.", "speedy.", 1), timeout=50, verify=False)
+        isSeries = "/series/" in self.novel_url
+        seriesLink = soup.select_one("a.bn_av")
+        self.novel_cover = soup.select_one("img.undefined")["src"] or None
```

Comment:
the `.undefined` class doesn't look reliable. if possible, i suggest using a better selector